### PR TITLE
Adding QuickNode section to the develope, adding resources, and updating QuickNode links.

### PR DESCRIPTION
**Pull request by Sahilsen**

### CODE REVIEW

File: docs/develop/quicknode.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+
+Explanation of the code above.
+
+Line 1: Specifying [SPDX license](https://spdx.org/licenses/) type, which is an addition after Solidity ^0.6.8.
+
+>Whenever the source code of a smart contract is made available to the public, these licenses can help resolve/avoid copyright issues. If you do not wish to specify any license type, you can use a special license [UNLICENSED](https://spdx.org/licenses/Unlicense.html) or simply skip the whole comment (it won't result in an error, just a warning).
+
+Line 2: On the first line, we are declaring which Solidity compiler we want to use. For instance, we are targeting any version between ≥ 0.4.0 and <0.7.0.
+
+Line 3: We are declaring our contract here and naming it as `Simplestorage`.
```
Comment (stephenplusplus) [inline docs/develop/quicknode.md:100]:
```suggestion
Line 3: We are declaring our contract here and naming it as `SimpleStorage`.
```

File: docs/develop/quicknode.md [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+
+Once the contract deployment transaction is approved, the deployed contract will appear under the `Deployed Contracts` section.
+
+![](https://lh3.googleusercontent.com/1Ble0P_qeM_xSA3o3qQNRjEpVl12z7eOUJsQjy7709p6b-5UlQNdaqZBgoUaVNjKiRlkYOaSPF-5S_S0unEBtCQuRH8YKBqxLD9PSMItVOOWAd3PGvr4F8WaGHE_2WFpkJYgoBVQ=s1600)
+
+Expand the deployed contract and click on `get`; it will return the value of the **storedData**, which is  currently zero since we have not input any number yet.
+
+![](https://lh6.googleusercontent.com/O4xP-eo65gqo2j7fb-FgECDxJXY0FONIjZghlapA_FhC5Hwxhf-QJIm1jiy1HDoQU7R_C_5h_W2JTelUqnmr6cdeRZFOWG9q5Iw_iZh94t-qgAoNwfJLjFND2XBwZtyQWuzODpxP=s1600)
+
+To input a value, enter a number in the field near the `set` button, click on set and approve the transaction from the MetaMask pop-up. Once the transaction is approved, value of the **storedData** will be the input number. To verify this, click on `get`, and the inputed value will be printed.
```
Comment (stephenplusplus) [inline docs/develop/quicknode.md:129]:
```suggestion
To input a value, enter a number in the field near the `set` button, click on set and approve the transaction from the MetaMask pop-up. Once the transaction is approved, value of the **storedData** will be the input number. To verify this, click on `get`, and the inputted value will be printed.
```

File: docs/develop/quicknode.md [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+
+![](https://lh3.googleusercontent.com/TXEC6gAAyFSmmZPKX9xUUGlJczLNAmtVb2MS6-yPR9dU-9XMef3KkcMCuCWTodfjpVgqL5cej7Ig93zsimSU8dF56KzTQEjPR9NIFAshCKlZSHCJFg0Gl9lxdy_BWG-QtWyWAS_m=s1600)
+
+So this is how contracts are deployed and interacted on Polygon.
+
+QuickNode just like Polygon has always had an education first approach they put out regular developer [guides](https://www.quicknode.com/guides?utm_source=polygon_docs&utm_campaign=ploygon_docs_contract_guide), [docs](https://www.quicknode.com/docs/polygon?utm_source=polygon_docs&utm_campaign=ploygon_docs_contract_guide), [tutorial videos](https://www.youtube.com/channel/UC3lhedwc0EISreYiYtQ-Gjg/videos) and have a [community of #web3 developers](https://discord.gg/DkdgEqE) who are eager to help eachother.
+
+We hope this guide was helpful. We'll be happy to hear from you. DM us or tag us on Twiiter [@QuickNode](https://twitter.com/QuickNode)
```
Comment (stephenplusplus) [inline docs/develop/quicknode.md:137]:
```suggestion
We hope this guide was helpful. We'll be happy to hear from you. DM us or tag us on Twitter [@QuickNode](https://twitter.com/QuickNode).
```

**Review by stephenplusplus — COMMENTED** [review]
Great guide!

**Comment by Sahilsen** [comment]
Hey, @stephenplusplus thank you so much for the review. I've changed the guide to use Brownie instead of REMIX as a REMIX guide was already in place as suggested by Nazim and Sandipan from Polygon, thanks.

Please check:
https://github.com/maticnetwork/matic-docs/pull/764/commits/790583c3ffde56b6ea400e6caf54a753746b43d4

File: docs/develop/quicknode.md [hunk 4]
```diff
@@ -0,0 +1,189 @@
...
+The contract is an ERC-20 contract; you can learn more about the ERC-20 standards and contracts in this [guide on ERC-20 tokens](https://www.quicknode.com/guides/solidity/how-to-create-and-deploy-an-erc20-token).
+
+## Step 2: Booting your Polygon node
+-------------------------
+
+QuickNode has a global network of Polygon Mainnet and Mumbai testnet nodes, today in this guide we will deploy our contract on the Polygon Mumbai testnet so sign up and simply get a [free trial node from QuickNode](https://www.quicknode.com/chains/matic?utm_source=polygon_docs&utm_campaign=ploygon_docs_contract_guide), which is much better than investing time in looking at different custom configs to launch your own node.
+
+![](https://lh6.googleusercontent.com/vEAp28OLizS-ZBoYU1yWthHalH0xbrgatZ1ynyA1H4y2tW-x1EHC97qk5RkHatq6tO2anLAC-ch4mfs4am2rf1zaMO5BNEAtL3anqL4UQVH5ebpQOKyItYLm4slWIurR9iznXkly=s1600)
+
+Copy the HTTP URL, which will be needed in the next step.
```
Comment (abunsen) [inline docs/develop/quicknode.md:101]: We should add a bit here that exposes the free/public RPC node QuickNode runs for Matic? Along with a note that says if they're rate limited to buy a node?

**Review by abunsen — CHANGES_REQUESTED** [review]

File: docs/develop/quicknode.md (hunk 4 above)
Comment (Sahilsen) [reply docs/develop/quicknode.md:101]: Done, added a line to mention the public RPC.

**Review by Nazeim — APPROVED** [review]

**Pull request merged**